autoload/clap/themes/srcery.vim

```diff
@@ -0,0 +1,137 @@
+" Author: Birger J. Nordolum  <contact+srcery@mindtooth.no>
+" Description: Clap theme based on the Srcery color scheme.  The theme
+" require some tweaks to make it work.  If you are to copy this code for
```

Comment:
Maybe write
> if you plan on copying or building upon this code for your own colorscheme, make sure...

instead of

> If you are to copy this code

Sounds a bit off, or stiff.